test/testcafe/spec/Interact_spec.js

```diff
@@ -114,6 +114,7 @@ async function setup(t, options = {}) {
     recoveryToken: options.recoveryToken,
     clientId: 'fake',
     redirectUri: 'http://doesnot-matter',
+    authScheme: 'oauth2',
     authParams: {
```

Comment:
Test was failing on Bacon because Bacon's widget config didn't have explicit `authScheme: 'oauth2'` so in `useOnSubmit` the `isOAuth2Enabled` check was not passing and the `recoveryToken` clearing logic was not getting hit